sale_stock_available_to_promise_release/models/sale_order_line.py

```diff
@@ -1,13 +1,73 @@
 # Copyright 2019 Camptocamp (https://www.camptocamp.com)
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
-from odoo import models
+from odoo import api, fields, models
+from odoo.tools import float_compare, float_is_zero
 
 
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    def _prepare_procurement_values(self, group_id=False):
-        values = super()._prepare_procurement_values(group_id)
-        values["date_priority"] = self.order_id.date_order
-        return values
+    availability_status = fields.Selection(
```

Comment:
Please add this in the order line before unit_price (with the three dot option that allow to hide it)